### PR TITLE
Remove deprecated Ember.K

### PR DESCRIPTION
**Pull request by cibernox**
`Ember.K` has been deprecated in Ember.

### CODE REVIEW

File: tests/unit/adapters/application-test.js [hunk 1]
```diff
@@ -940,7 +940,7 @@ test('#fetch handles 200 (Success) response status', function(assert) {
   const adapter = this.subject({type: 'posts', url: '/posts'});
   mockFetchJax(sandbox, adapter, postMock);
   sandbox.stub(adapter, 'cacheResource', function () {});
-  adapter.serializer = { deserialize: sandbox.spy(), deserializeIncluded: Ember.K };
+  adapter.serializer = { deserialize: sandbox.spy(), deserializeIncluded() {} };
```
Comment (pixelhandler) [inline tests/unit/adapters/application-test.js]: @cibernox nice, thanks for the fix. So named function declaration in shorthand syntax . Sweet. `deserializeIncluded() {}` and clean

**Comment by pixelhandler** [comment]
@cibernox looks like the Travis build fails. I'll take a look over the weekend.

**Comment by cibernox** [comment]
Actually this PR was made with an automatic tool that does all the transformations automatically https://github.com/cibernox/ember-k-codemod

I'm working on a script that it's going to fix every open source addon automatically, because after 3/4 pull requests this is boring.

**Comment by pixelhandler** [comment]
@cibernox I've been interested in creating code mods for things like blue print upgrades, e.g. many files generated with a blueprint and then upgrading the blueprint and providing a codemode to migration existing (generated) files.

**Comment by cibernox** [comment]
It is fun indeed, but beware, documentation is scarce. It's mostly a matter of trial and error (and a lot of debuggers)

**Comment by pixelhandler** [comment]
@cibernox one way "I cut my teeth" (learning) programming was writing code mods for phpMyBB, the format was like find `code` and replace with `new code`. before learning to write an app/system I learned how to hack it for customized behaviors. Seems like a nice modx like tool could be great for patching existing ember apps.

**Comment by pixelhandler** [comment]
@cibernox I added a commit to fixup an unused Ember import in a test.

File: tests/unit/mixins/resource-operations-test.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import Ember from 'ember';
```
Comment (pixelhandler) [inline tests/unit/mixins/resource-operations-test.js]: @cibernox after removing Ember.K the import was no longer needed.

**Pull request merged**